### ISSUE TITLE
Using `font: var(...)` overwrites use of `font-feature-settings`

### ISSUE THREAD

**Issue by Barsnes** [digdir/designsystem#922]
### Description of the bug

It is not possible to set `font-feature-settings: ...` without using `!important`.
This is because we set the font on components using `font: ...`, which overwrites the use on upper selectors.

### Steps To Reproduce

Set this in a css file:
```css
font-feature-settings: 'cv05' 1;
```

and then this on any component:
```css
font: var(--fds-typography-paragraph-small)
```

The setting we turned on should show lowercase L with a tail, but it does not.

### Additional Information

We should figure out if there is a better way to do `font: ...`.
Maybe css classes where we don't use `font-feature-settings` would be an options?

**Comment by Barsnes** [comment]
After reading about [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font), I found that
> font-style, font-variant and font-weight must precede font-size
This means that if we write our variables like this example, we will fix this issue:
`unset 400 clamp(1.00rem,calc(0.19vw + 0.96rem),1.13rem)/1.5"Inter"`
This just tells the browser that `-style` and `-variant` is unset!

**Comment by Barsnes** [comment]
> After reading about [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font), I found that
> 
> > font-style, font-variant and font-weight must precede font-size
> 
> This means that if we write our variables like this example, we will fix this issue: `unset 400 clamp(1.00rem,calc(0.19vw + 0.96rem),1.13rem)/1.5"Inter"` This just tells the browser that `-style` and `-variant` is unset!

This did not work, and referencing [font on mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/font):
> As with any shorthand property, any individual value that is not specified is set to its corresponding initial value (possibly overriding values previously set using non-shorthand properties). Though not directly settable by font, the longhands [font-size-adjust](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size-adjust) and [font-kerning](https://developer.mozilla.org/en-US/docs/Web/CSS/font-kerning) are also reset to their initial values.

**Comment by mimarz** [comment]
Document in tokens README use that typography css variables are meant to be used with `font`

**Comment by mrosvik** [comment]
Will be fixed by [#957](https://github.com/digdir/designsystem/issues/957)
